t/compile.t

```diff
@@ -18,11 +18,8 @@ map {
     }, catfile($thisdir, $_));
 } qw(../bin ../lib);
 
-for (@tests) {
-    my $output = `perl -I "$libpath" -c $_ 2>&1`;
-    my $ok = ($? >> 8 == 0);
-    print $output unless $ok;
-    ok($ok, "$_ syntax check");
+foreach my $file (@tests) {
+    require_ok $file;
```

Comment:
> Why wouldn't require_ok be using your Perl path? I did notice $libpath, but that's needed because you're launching a new subprocess. require_ok runs in process and should have access to the same INC that t/compile.t has access to

This is because tests don't setup @INC [the way `bin/serge` does](https://github.com/evernote/serge/blob/master/bin/serge#L5-L10). Remember that Serge may run from any directory, even not being formally 'installed'.